src/bicep-registry/scheduled-action/main.bicep

```diff
@@ -0,0 +1,138 @@
+// @resourceGroup @subscription @tenant
+
+// See https://learn.microsoft.com/rest/api/cost-management/scheduled-actions/create-or-update
+
+//===| Parameters |============================================================
+
+targetScope = 'subscription'
+
+@description('Name of the scheduled action used in the resource ID.')
+param name string
+
+@description('Indicates the kind of scheduled action.')
+@allowed([
+  'Email'
+  'InsightAlert' // @subscription
+])
+param kind string = 'Email'
+
+// @tenant
+//   @description('The Azure resource ID for the billing scope to use. This is the full resource ID of a billing account, billing profile, department, etc. Do not use the GUID only.')
+//   param scope string
+
+@description('Specifies which built-in view to use. This is a shortcut for the full view ID.')
+@allowed([
+  ''
+  'AccumulatedCosts'
+  'DailyCosts'
+  'InvoiceDetails'
+])
+param builtInView string = ''
+
+@description('Required if kind is "Email" and builtInView is not set. The resource ID of the view to which the scheduled action will send. The view must either be private (tenant level) or owned by the same scope as the scheduled action. Ignored if kind is "InsightAlert" or if builtInView is set.')
+param viewId string = ''
+
+@description('The display name to show in the portal when viewing the list of scheduled actions.')
+param displayName string = name
+
+@description('The status of the scheduled action.')
+@allowed([
+  'Enabled'
+  'Disabled'
+])
+param status string = 'Enabled'
+
+@description('Email address of the person or team responsible for this scheduled action. This email address will be included in emails.')
+param notificationEmail string = ''
+
+@description('List of email addresses that should receive emails. At least one valid email address is required.')
+param emailRecipients array
+
+@description('The subject of the email that will be sent to the email recipients.')
+@maxLength(70)
+param emailSubject string = ''
+
+@description('Include a message for recipients to add context about why they are getting the email, what to do, and/or who to contact.')
+@maxLength(250)
+param emailMessage string = ''
+
+@description('The language that will be used for the email template.')
+param emailLanguage string = 'en'
+
+@description('The regional format that will be used for dates, times, and numbers.')
+param emailRegionalFormat string = 'en-US'
+
+@description('Indicates whether to include a link to a CSV file with the backing data for the chart. Ignored if kind is "InsightAlert".')
+param includeCsv bool = false
```

Comment:
Should we have options?
csv, csv+gzip, csv+zip(though this should be default), parquet, etc.

Reply:
Keep in mind the scenario here is more for end users, not systems, so I'm not sure how critical parquet is. That said, when we support webhooks, it would likely be. I'd love to find a way to make this consistent across all APIs.